torch/testing/_internal/common_methods_invocations.py

```diff
@@ -3903,6 +3916,12 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
            supports_inplace_autograd=False,
            supports_out=False,
            sample_inputs_func=sample_inputs_trace),
+    OpInfo('kron',
+            dtypes=all_types_and_complex_and(torch.bool, torch.half, torch.bfloat16),
+            dtypesIfCUDA=all_types_and_complex_and(torch.bool, torch.half, torch.bfloat16),
+            supports_inplace_autograd=False,
+            supports_out=False,
```

Comment:
kron does support out, so I think this line can be removed?

Reply:
Done.